src/App.js

```diff
@@ -1,41 +1,24 @@
 import "./App.css";
 import logo from "./logo.png";
-
+import {Routes,Route} from "react-router-dom";
+import Home from "./Pages/Home/Home";
+import Product from "./Pages/Product/Product";
+import Wishlist from "./Pages/Wishlist/Wishlist";
+import Cart from "./Pages/Cart/Cart";
+import Login from "./Pages/Auth/Login";
+import Signup from "./Pages/Auth/Signup";
```

Comment:
Suggestion: by using barrel import all these components can be Imported in one line from `./Pages`

Reply:
This is new way. ok will sure do this.